tests/integration/src/test/java/org/mule/context/notification/FlowStackTestCase.java

```diff
@@ -38,358 +38,409 @@ protected String getConfigFile()
         return "org/mule/test/integration/notifications/flow-stack-config.xml";
     }
 
+    private static final String TEST_MESSAGE = "payload";
```

Comment:
There is a TEST_MESSAGE constant already defined somewhere in the test hierarchy. Re use that one